src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioAnalyzer.cs

```diff
@@ -135,5 +136,37 @@ public Assembly LoadFromPath(string fullPath)
                 return _analyzer._loader.LoadFromPath(fullPath);
             }
         }
+
+        /// <summary>
+        /// This custom <see cref="AnalyzerReference"/>, just wraps an existing <see cref="UnresolvedAnalyzerReference"/>,
```

Comment:
For some reason the word "existing" here threw me for a loop -- it's still the thing creating the underlying one too.